utils/build/docker/golang/app/net-http.go

```diff
@@ -134,13 +146,120 @@ func main() {
 		appsec.TrackCustomEvent(r.Context(), name, map[string]string{"metadata0": "value0", "metadata1": "value1"})
 	})
 
+	mux.HandleFunc("/e2e_otel_span", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can you add a parapgraph in `docs/weblog/README.md` explaining what this endpoint does ?